spec/rspec/support/ruby_features_spec.rb

```diff
@@ -145,7 +145,7 @@ def ripper_can_parse_source_including_keywordish_symbol?
         end
 
         it 'returns whether Ripper is correctly implemented in the current environment' do
-          expect(RubyFeatures.ripper_supported?).to eq(ripper_is_implemented? && ripper_works_correctly?)
+          expect(RubyFeatures.ripper_supported?).to eq(RubyFeatures.ripper_supported? && ripper_is_implemented? && ripper_works_correctly?)
```

Comment:
```ruby
expect(RubyFeatures.ripper_supported?).to eq(RubyFeatures.ripper_supported?
```
That looks odd quite frankly :D

Reply:
This change should be reverted, as `ripper_supported?` will be `false`, the test methods should return false as ripper isn't implemented properly on truffle right?

Reply:
This should be reverted, the check here is that `ripper_supported?` matches the two test definitions, are you saying the test definitions return `true` here? As that should be impossible if ripper is broken on truffleruby. Whats actually happening?

Reply:
@JonRowe On `truffleruby`, a runtime error occurs when running `ripper_is_implemented? && ripper_works_correctly?` which is why I suggested the `begin/rescue` change before to assert that running these results in error when `RubyFeatures.ripper_supported?` is `false`.

Per your comment: https://github.com/rspec/rspec-support/pull/395#discussion_r365361330. I updated to not run this spec if `RubyFeatures.ripper_supported?` is `false`.

@pirj I agree this looked odd. I've pushed a new version to be more explicit in excluding this spec from running using: `, :if => ripper_supported?`. Please review.